pkg/apiserver/handlers_test.go

```diff
@@ -151,6 +151,15 @@ func TestGetAPIRequestInfo(t *testing.T) {
 		{"GET", "/namespaces", "list", "", "", "namespaces", "", "Namespace", "", []string{"namespaces"}},
 		{"GET", "/namespaces/other", "get", "", "other", "namespaces", "", "Namespace", "other", []string{"namespaces", "other"}},
```

Comment:
If this is still passing, then my previous question about /namespaces/foo having a context of foo is answered that for a namespaced resource, the context is foo.  This means that there could be a problem when updating a namespace based on what I read in update.go above.  Need to check if/how that is handled or if I am missing something.